assets/js/blocks/catalog-sorting/edit.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { useBlockProps } from '@wordpress/block-editor';
+import { Disabled } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block';
+
+interface Props {
+	attributes: {
+		className?: string;
+	};
+}
+
+const Edit = ( { attributes }: Props ): JSX.Element => {
+	const { className } = attributes;
+	const blockProps = useBlockProps( {
+		className: classNames( 'wc-block-catalog-sorting', className ),
+	} );
+
+	return (
+		<div { ...blockProps }>
+			<Disabled>
+				<Block />
+			</Disabled>
+		</div>
+	);
+};
+
+export default Edit;
```

Comment:
Because we don't have any custom attribute, this can be simplified to

```suggestion
/**
 * External dependencies
 */
import { useBlockProps } from '@wordpress/block-editor';
import { Disabled } from '@wordpress/components';

/**
 * Internal dependencies
 */
import Block from './block';

const Edit = () => {
	const blockProps = useBlockProps( {
		className: 'wc-block-catalog-sorting',
	} );

	return (
		<>
			<div { ...blockProps }>
				<Disabled>
					<Block />
				</Disabled>
			</div>
		</>
	);
};

export default Edit;
```

Reply:
👍